### PR TITLE
remove trademark from Qiskit and update readme

### PR DESCRIPTION
**Pull request by abbycross**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -599,9 +599,12 @@ Some companies require a special attribution notice. View a list of the companie
 - IBM&reg;
 - IBM Cloud&reg;
 - IBM Quantum&trade;
-- Qiskit&reg;
 </details>
 
+<Admonition type="note" title="Qiskit is an exception">
```
Comment (Eric-Arellano) [inline README.md:604]: This doesn't work in GitHub's README, which doesn't have MDX components.

File: README.md [hunk 2]
```diff
@@ -614,4 +617,4 @@ use `&trade;` to get &trade; for nonregistered trademarks.
 
 <Admonition type="caution">
```
Comment (Eric-Arellano) [inline README.md:618]: My bad for not catching this earlier.

**Review by Eric-Arellano — APPROVED** [review]
Thank you!

File: README.md (hunk 1 above)
Comment (abbycross) [reply README.md:604]: haha oops! I'll remove those

**Review by Eric-Arellano — APPROVED** [review]

**Pull request merged**